readme.md

```diff
@@ -53,23 +58,23 @@ var socket = io('http://localhost:3000');
 var uploader = new SocketIOFileClient(socket);
 var form = document.getElementById('form');
 
-uploader.on('start', function(fileInfo) {
+uploader.on('start', (fileInfo) => {
```

Comment:
This breaks the compatibility with IE11

Reply:
This is modern JS. Compatibility should be archived by compilation with Webpack, Gulp, Babel, etc.

Reply:
@tuxpoldo Well, he is correct. This module is designed to support UMD, and also supports the old JavaScript environment which loads the script via <script> tag.

Reply:
I will revert this changes then. Are the interfaces correct tho?